clang/include/clang/Parse/Parser.h

```diff
@@ -5188,6 +5188,14 @@ class Parser : public CodeCompletionHandler {
       ParseHLSLAnnotations(Attrs, EndLoc);
   }
 
+  struct ParsedSemantic {
+    StringRef Name;
```

Comment:
Based on the implementation of `ParseHLSLSemantic` all these fields have default values, you should use in class member initializers and just return the defaulted values instead of manually setting the same values three times. This will be very error prone under refactoring and likely won't remain consistent.